homeassistant/components/goodwe/const.py

```diff
@@ -0,0 +1,26 @@
+"""Constants for the Goodwe component."""
+from datetime import timedelta
+
+from homeassistant.components.number import DOMAIN as NUMBER_DOMAIN
+from homeassistant.components.select import DOMAIN as SELECT_DOMAIN
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+
+DOMAIN = "goodwe"
+
+PLATFORMS = [NUMBER_DOMAIN, SELECT_DOMAIN, SENSOR_DOMAIN]
+
+DEFAULT_NAME = "GoodWe"
+SCAN_INTERVAL = timedelta(seconds=10)
+
+CONF_MODEL_FAMILY = "model_family"
+
+KEY_INVERTER = "inverter"
+KEY_COORDINATOR = "coordinator"
+KEY_DEVICE_INFO = "device_info"
+
+INVERTER_OPERATION_MODES = [
```

Comment:
Since we're only using these modes in the number platform it's easier to read if the constant is defined in that module. Normally we only put common constants in const.py.

Reply:
Done